feedback/static/feedback.css

```diff
@@ -338,6 +338,11 @@ body {
 		justify-content: flex-start;
 	}
 }
+.feedback-response-panel > .panel-body > .conversation-tag-panel > .colortag:not(:hover):not(.colortag-active) {
```

Comment:
The usability would be significantly improved already if this is changed to:
@media (hover: hover) {
	.feedback-response-panel > .panel-body > .conversation-tag-panel:not(:hover) > .colortag:not(.colortag-active) {
...
	}
}